numpy/lib/function_base.py

```diff
@@ -1027,9 +1027,11 @@ def average(a, axis=None, weights=None, returned=False):
     a : array_like
         Array containing data to be averaged. If `a` is not an array, a
         conversion is attempted.
-    axis : int, optional
-        Axis along which to average `a`. If `None`, averaging is done over
-        the flattened array.
+    axis : int or tuple of ints, optional
+        Axis or axes along which to average `a`. If `None`, averaging is done
+        over the flattened array.
+        If this is a tuple of ints, averaging is performed over multiple axes,
```

Comment:
I copied this from: https://github.com/numpy/numpy/blob/eefc4b8bea043a324927e40d998ed760799f0b82/numpy/core/fromnumeric.py#L2803-L2810.

Do you like the mean formatting or the sum formatting?

I wasn't sure of what version added to use. I can use `.. versionadded:: 1.7.0` if you want.